index.html

```diff
@@ -20,8 +20,10 @@
 
 
         <script id="email-catalog-template" type="text/x-handlebars-template">
+            <button>Mark As Read</button>
```

Comment:
I think we should add an id here to better select it. This prevents us from having multiple buttons